### ISSUE TITLE
Send Keys is sending keys out side the container, But working in interactive mode via VNC

### ISSUE THREAD

**Issue by piyugit** [zalando/zalenium#956]
Hi ,
My methos for file upload is (C#,.net 4.7 with selenium)

 public void UploadPdfTemplate(string filePath)
        {
            _parallelConfig.Driver.WaitForAction();
            SendKeys.SendWait(filePath);
            SendKeys.SendWait(@"{Enter}");
            _parallelConfig.Driver.WaitForAction(1);
        }
Working in local but on Zalenium container in parallel/Serial execution it is sending event out side the container.
Note:- If I Interact via VNC its working.

**Comment by diemol** [comment]
Hi @piyugit,

How can I reproduce that? Can you please create a new issue and fill out the complete template?
